apps/demo/translations/src/en/fields.json

```diff
@@ -3,13 +3,16 @@
     "legend": "Which form would you like to explore?",
     "options": {
       "basic-form": {
-        "label": "Basic form"
+        "label": "Basic form",
+        "hint": "An example of a simple form produced using the hof framework. This form demos common components used by services using the hof framework."
```

Comment:
Could you uppercase the "HOF"? I'd also maybe put that it "showcases components and features" available in the HOF framework" instead.

Reply:
changed hof to HOF and have altered the text.